src/main/java/com/mongodb/jdbc/MySQLFunctions.java

```diff
@@ -0,0 +1,730 @@
+// This is generated code. To regenerate go to the resources directory:
```

Comment:
This comment seems old, is this file still generated code?

Reply:
Yes, not sure where I got this from. Certainly does not apply here.